docs/tuf-spec.txt

```diff
@@ -731,12 +742,27 @@ Version 1.0 (Draft)
            "keyids" : [ KEYID, ... ] ,
            "threshold" : THRESHOLD,
            ("path_hash_prefixes" : [ HEX_DIGEST, ... ] |
-            "paths" : [ PATHPATTERN, ... ])
+            "paths" : [ PATHPATTERN, ... ]),
+           "terminating": TERMINATING,
        }, ... ]
      }
-     
-   ROLENAME is the full name of the delegated role.  For example,
-   "targets/projects"
+
+   ROLENAME is the name of the delegated role.  For example,
+   "projects".
```

Comment:
[Section 3.1.2.1](https://github.com/theupdateframework/tuf/blob/3532fb800533f64484092fffa97a376ecc130c73/docs/tuf-spec.txt#L412-L427) explains how delegated metadata is placed on the repository.

This change is certainly backwards-incompatible, but so are many of the changes in this version 1.0 of the specification (e.g., terminating delegations, version numbers in Snapshot, etc.)

I'm not sure that we can set restrictions on the rolename, which must ultimately match the filename of the role as saved to the underlying file system.  Should we state in the specification that we do not place restrictions on rolenames, and supported characters in rolenames are left to adopters?